src/components/Menu/MenuItem/__stories__/MenuItem.stories.mdx

```diff
@@ -67,6 +70,12 @@ Use menu item for drawing one options that displayed inside a menu.
   <Story name="Icons with colors">{menuItemIconsWithColorsTemplate.bind({})}</Story>
 </Canvas>
 
+### Label
+
+<Canvas>
+  <Story name="Label">{menuItemLabelTemplate.bind({})}</Story>
```

Comment:
Is this inlined correctly in the "Show code" section? are you able to see the actual code?

Reply:
Yes it is shown